plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -118,6 +124,7 @@ func newQuotaEvaluator(client clientset.Interface, registry quota.Registry) (*qu
 		registry:        registry,
 		liveLookupCache: liveLookupCache,
 		liveTTL:         time.Duration(30 * time.Second),
+		updatedQuotas:   map[string]*api.ResourceQuota{},
```

Comment:
does this map grow unbounded?  we never seem to expire anything from this cache.